compose/cli-command-compatibility.md

```diff
@@ -0,0 +1,49 @@
+---
```

Comment:
This page appears as an orphan page - https://deploy-preview-12433--docsdocker.netlify.app/compose/cli-command-compatibility/. It must be linked from the toc.

Reply:
I thought linking it from the compose tech preview page was OK. Is it mandatory to make it appear in the TOC. (fine for me if we prefer to link everything in TOC)

Reply:
Yes, we should ideally link it from the toc. If users land on this page, there's no context if they want to go back to a previous page, or select a different topic.

Reply:
Added `Compose Tech preview compatibility` section just next to `Compose CLI Tech Preview`

Reply:
Yea, agree. We can add that as a subsection.